src/Confluent.Kafka/Timestamp.cs

```diff
@@ -26,6 +26,16 @@ namespace Confluent.Kafka
     /// </summary>
     public struct Timestamp
     {
+        private const long RD_KAFKA_NO_TIMESTAMP = 0;
+
+        /// <summary>
+        ///     A read-only field representing an unspecified timestamp.
+        /// </summary>
+        public static Timestamp Default
```

Comment:
I'm undecided on what to call this. `Default` works well when specifying this in a `Produce` or `ProduceAsync` call, but `NotSpecified` may be better. Where else does this value show up?

Reply:
I was thinking about that. I didn't like that it possibly implies that these are two different values and that you could technically compare the timestamp in a delivery report to something called 'Now' and get true (when it is unavailable). That said, it's probably unlikely to confuse and maybe the best tradeoff. I plan to agonize over this decision further for some overly long amount time.

Reply:
I keep coming back to `.Default` being the preferred compromise here, don't plan to change.